### PR TITLE
evolution-data-server: update to 3.30.5.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/evolution-data-server/template [hunk 1]
```diff
@@ -16,7 +16,7 @@ license="LGPL-2.1-only"
 homepage="https://wiki.gnome.org/Apps/Evolution"
 changelog="https://raw.githubusercontent.com/GNOME/evolution-data-server/gnome-3-30/NEWS"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=de9cc30bdfaeff9dea561d7d1d43f31dfd24d0fdf3c231cfd3a9fc4f00e514c9
+checksum=91d6312f60752bb5229e35975b3c1f30f8f64bd8e5cce76cc275b1073d1725e9
 
 build_options="gir"
```
Comment (maxice8) [inline srcpkgs/evolution-data-server/template]: If possible enable gir for this package :D

Comment (Cogitri) [reply srcpkgs/evolution-data-server/template]: Will do

Comment (Cogitri) [reply srcpkgs/evolution-data-server/template]: Alright, this might take a little longer, how about merging this now and enabling GIR sometime later?

**Comment by maxice8** [comment]
On Sat, Feb 09, 2019 at 06:37:57AM -0800, Rasmus Thomsen wrote:
> Cogitri commented on this pull request.
> 
> 
> 
> > @@ -16,7 +16,7 @@ license="LGPL-2.1-only"
>  homepage="https://wiki.gnome.org/Apps/Evolution"
>  changelog="https://raw.githubusercontent.com/GNOME/evolution-data-server/gnome-3-30/NEWS"
>  distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
> -checksum=de9cc30bdfaeff9dea561d7d1d43f31dfd24d0fdf3c231cfd3a9fc4f00e514c9
> +checksum=91d6312f60752bb5229e35975b3c1f30f8f64bd8e5cce76cc275b1073d1725e9
>  
>  build_options="gir"
> 
> Alright, this might take a little longer, how about merging this now and enabling GIR sometime later?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8214#discussion_r255300578

sure

**Pull request merged**